assets/js/modules/analytics/components/module/ModuleOverviewWidget/index.js

```diff
@@ -74,29 +76,46 @@ export default function ModuleOverviewWidget( { Widget, WidgetReportError } ) {
 	const statsReport = useSelect( ( select ) => select( MODULES_ANALYTICS ).getReport( statsArgs ) );
 	const statsError = useSelect( ( select ) => select( MODULES_ANALYTICS ).getErrorForSelector( 'getReport', [ statsArgs ] ) );
 
-	if ( overviewError || statsError ) {
-		return <WidgetReportError statsError={ overviewError || statsError } />;
+	const isZero = isZeroReport( overviewReport );
+	const isError = overviewError || statsError;
+
+	if ( ! overviewLoaded || ! statsLoaded ) {
+		return <ProgressBar />;
```

Comment:
Let's use `PreviewBlock` here. While the legacy component here relies on `ProgressBar` and this is indeed a change, we should build the new widget implementation consistent with how the widgets in the dashboard work. It'd be better to then also wrap that `PreviewBlock` in the regular `Widget` UI below.

Reply:
Updated.